rpm_spec/core-agent.spec.in

```diff
@@ -582,6 +582,10 @@ if [ -L /usr/local ]; then
     mount /usr/local || :
 fi
 
+%if 0%{?fedora} < 41
+dnf config-manager --setopt=gpgcheck=1 --setopt=zchunk=0 --setopt=deltarpm=0 --save
```

Comment:
Maybe add some flag file to avoid re-running it over and over (possibly overriding user's explicit choice to change any of those)?
And the same comment for `gpgcheck=1`.